src/utils.rs

```diff
@@ -483,6 +483,11 @@ pub fn render_time(raw_millis: u128, show_millis: bool) -> String {
         .collect();
     if show_millis || raw_millis < 1000 {
         rendered_components.push(render_time_component((&millis, &"ms")));
+
+        // Default component
```

Comment:
I think you got it right. We basically have to either hardcode a special case in render_time_component or hardcode a special case into render_time. Since the special case is more applicable to render_time, might as well put it there.